### PR TITLE
chore(ajax): convert demos to new storybook

### PR DESCRIPTION
**Pull request by jorenbroekema**

### CODE REVIEW

File: packages/ajax/stories/index.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    console.log(response.data);
+  })
+  .catch(error => {
+    console.log(error.message);
+  });
+```
+
+## Considerations
+
+Due to a [bug in axios](https://github.com/axios/axios/issues/385) options may leak in to other instances.
```
Comment (CubLion) [inline packages/ajax/stories/index.stories.mdx]: Links are rendered weird:

```html
<a href="http://localhost:9002https://github.com/axios/axios/issues/385" class="sbdocs sbdocs-a css-1cvpre6">bug in axios</a>
```

Comment (jorenbroekema) [reply packages/ajax/stories/index.stories.mdx]: Odd.. will need to look into how to prevent that in storybook .mdx files

Comment (jorenbroekema) [reply packages/ajax/stories/index.stories.mdx]: https://github.com/storybookjs/storybook/issues/9005 I think this is related

**Comment by CubLion** [comment]
After https://github.com/ing-bank/lion/pull/446/files#r360317791 LGTM

**Pull request merged**